model/src/main/resources/db-update/20060-from-2.5.0-to-2.6.0.sql

```diff
@@ -24,61 +24,62 @@ BEGIN;
+CREATE TABLE deliverableanalyzerreport
+(
+    operation_id        BIGINT                  NOT NULL,
+    labels              TEXT,
+    PRIMARY KEY (operation_id)
+);
+ALTER TABLE deliverableanalyzerreport
+    ADD CONSTRAINT fk_deliverableanalyzerreport_operation
+        FOREIGN KEY (operation_id)
             REFERENCES operation(id);
 
 --------------------------------------------------------------------------------
 -- DeliverableArtifact
 --------------------------------------------------------------------------------
-CREATE TABLE deliverable_artifact
-    (
-        report_id           BIGINT                  NOT NULL,
-        artifact_id         INTEGER                 NOT NULL,
-        built_from_source   BOOLEAN                 NOT NULL,
-        brew_build_id       INTEGER,
-        PRIMARY KEY (report_id, artifact_id)
-    );
-    ALTER TABLE deliverable_artifact
-        ADD CONSTRAINT fk_deliverableartifact_report
-            FOREIGN KEY (report_id)
-            REFERENCES delan_report(operation_id);
-    ALTER TABLE deliverable_artifact
-        ADD CONSTRAINT fk_deliverableartifact_artifact
-            FOREIGN KEY (artifact_id)
+CREATE TABLE deliverableartifact
+(
+    report_id           BIGINT                  NOT NULL,
+    artifact_id         INTEGER                 NOT NULL,
+    builtfromsource     BOOLEAN                 NOT NULL,
+    brewbuildid         BIGINT,
+    PRIMARY KEY (report_id, artifact_id)
+);
+ALTER TABLE deliverableartifact
+    ADD CONSTRAINT fk_deliverableartifact_delanreport
+        FOREIGN KEY (report_id)
+            REFERENCES deliverableanalyzerreport(operation_id);
+ALTER TABLE deliverableartifact
+    ADD CONSTRAINT fk_deliverableartifact_artifact
+        FOREIGN KEY (artifact_id)
             REFERENCES artifact(id);
 
 --------------------------------------------------------------------------------
 -- DeliverableAnalyzerLabelEntry
 --------------------------------------------------------------------------------
-CREATE TABLE delan_label_entry
-    (
-        id                  INTEGER                 NOT NULL,
-        report_id           BIGINT,
-        order_id            INTEGER,
-        entry_time          TIMESTAMP,
-        user_id             INTEGER,
-        reason              TEXT,
-        delan_report_label  TEXT,
-        change              TEXT,
-        PRIMARY KEY (id)
-    );
-    ALTER TABLE delan_label_entry
-        ADD CONSTRAINT uk_reportid_orderid
-            UNIQUE (report_id, order_id);
-    ALTER TABLE delan_label_entry
-        ADD CONSTRAINT fk_delanlabelentry_report
-            FOREIGN KEY (report_id)
-            REFERENCES delan_report(operation_id);
-    ALTER TABLE delan_label_entry
-        ADD CONSTRAINT fk_delanlabelentry_user
-            FOREIGN KEY (user_id)
+CREATE TABLE deliverableanalyzerlabelentry
+(
+    id              INTEGER                 NOT NULL,
+    report_id       BIGINT,
+    changeorder     INTEGER,
```

Comment:
This is mapped as "change_order" in https://github.com/project-ncl/pnc/blob/v2.6.x/model/src/main/java/org/jboss/pnc/model/DeliverableAnalyzerLabelEntry.java#L71.